### PR TITLE
(1/17) Creates `XRPAccountSet` Protobuf Wrapper Class

### PR DESCRIPTION
**Pull request by amiecorso**
## High Level Overview of Change
This PR implements the class `XRPAccountSet` - a native Typescript class to wrap the [`AccountSet` protocol buffer](https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L100).  See also: https://xrpl.org/accountset.html.

### Context of Change
Effort to implement additional [XRPL transaction types](https://xrpl.org/transaction-types.html) beyond `Payment` in the SDK.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

## Before / After
`XRPAccountSet` class and conversion unit tests now exist.

## Test Plan
Unit tests added to verify conversion.  CI passes.

## Future Tasks

Remaining transaction types & all three SDKs.
Implementation and exposure of client methods for submitting additional transaction types.

### CODE REVIEW

File: src/XRP/model/xrp-account-set.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { AccountSet } from '../Generated/web/org/xrpl/rpc/v1/transaction_pb'
+
+/*
+ * Represents an AccountSet transaction on the XRP Ledger.
+ * An AccountSet transaction modifies the properties of an account in the XRP Ledger.
```
Comment (keefertaylor) [inline src/XRP/model/xrp-account-set.ts:5]: nit: +1 blank line

File: src/XRP/model/xrp-account-set.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      emailHash,
+      messageKey,
+      setFlag,
+      transferRate,
+      tickSize,
+    )
+  }
+
+  /**
+   *
```
Comment (keefertaylor) [inline src/XRP/model/xrp-account-set.ts:38]: nit: extra blank line

File: src/XRP/model/xrp-account-set.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * @see: https://xrpl.org/accountset.html
+ */
+export default class XRPAccountSet {
+  /**
+   * Constructs an XRPAccountSet from an AccountSet.
+   *
+   * @param accountSet an AccountSet (protobuf object) whose field values will be used
+   *                to construct an XRPAccountSet
+   * @return an XRPAccountSet with its fields set via the analogous protobuf fields.
+   * @see https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L100
```
Comment (tedkalaw) [inline src/XRP/model/xrp-account-set.ts:15]: are we doing `-` between doc @params etc?

Comment (amiecorso) [reply src/XRP/model/xrp-account-set.ts:15]: we haven't done in any of our other `XRPWhatever` model objects... so for now I'm going to leave it like this.  When the ESLint configurations that Hans has been testing out on `xpring-common-js`, then I suspect we'll do a massive linting overhaul all at once.

**Pull request merged**